src/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -196,11 +205,10 @@ export const PersistentFilters = ({
 }: PersistentFiltersProps &
 	Required<Pick<PersistentFiltersProps, 'renderMode'>>) => {
 	const { state: analytics } = useAnalyticsContext();
-	const { pathname } = useLocation();
-	const locationSearch = history?.location?.search ?? '';
+	const { pathname, search } = document.location;
 	const storedFilters = React.useMemo(() => {
-		return loadRulesFromUrl(locationSearch, schema, history);
-	}, [locationSearch, schema, history]);
+		return loadRulesFromUrl(search, schema, history);
+	}, [search, schema, history]);
```

Comment:
`search` can be removed since `document.location` is not a reactive source

Reply:
These are not document.search and pathname, these are from `useLocation`